lib/internal/modules/cjs/loader.js

```diff
@@ -928,6 +928,13 @@ Module._load = function(request, parent, isMain) {
       delete Module._cache[filename];
       if (parent !== undefined) {
         delete relativeResolveCache[relResolveCacheIdentifier];
+        const children = parent && parent.children;
+        if (children) {
```

Comment:
Suggestion:
```suggestion
        if (ArrayIsArray(children)) {
```
For robustness - a module can do `module.children = null`.

Reply:
nice!  Array.isArray done

Reply:
fixed, thanks!